resteasy/src/test/scala/com/cerner/beadledom/resteasy/ModuleProcessorSpec.scala

```diff
@@ -0,0 +1,68 @@
+package com.cerner.beadledom.resteasy
+
+import org.jboss.resteasy.spi.{Registry, ResteasyProviderFactory}
+import org.scalatest.{FunSpec, MustMatchers}
+import org.scalatest.mockito.MockitoSugar
+import com.cerner.beadledom.guice.dynamicbindings.AnnotatedModule
+import com.google.inject.Guice
+
+class ModuleProcessorSpec extends FunSpec with MockitoSugar with MustMatchers{
+
+  val registry: Registry = mock[Registry]
+  val providerFactory: ResteasyProviderFactory = mock[ResteasyProviderFactory]
+
+  describe("ModuleProcessor#new") {
```

Comment:
I think in other tests we use nested `describe` blocks for the class methods.

```scala
describe("ModuleProcessor") {
  describe("constructor") {
  }

  describe("#processInjector") {
  }
}
```

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8